lms/static/js/spec/main.js

```diff
@@ -181,22 +181,7 @@
             },
             'underscore': {
                 deps: ['underscore.string'],
-                exports: '_',
-                init: function(UnderscoreString) {
```

Comment:
Also straight from [`underscore.string`](https://github.com/epeli/underscore.string)'s README:

"But [mixing in underscore.string into _] is not recommended since include, contains, reverse and join are dropped because they collide with the functions already defined by Underscore.js."